### PR TITLE
Add autoware_msgs to humble, jazzy, and rolling indices

### PR DESCRIPTION
**Pull request by youtalk**
# Please Add This Package to be indexed in the rosdistro.

humble, jazzy, and rolling

# The source is here:

https://github.com/autowarefoundation/autoware_msgs.git

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: humble/distribution.yaml [hunk 1]
```diff
@@ -652,6 +652,12 @@ repositories:
       url: https://github.com/autowarefoundation/autoware_cmake.git
       version: rolling
     status: developed
+  autoware_msgs:
+    source:
+      type: git
+      url: https://github.com/autowarefoundation/autoware_msgs.git
```
Comment (clalancette) [inline humble/distribution.yaml:658]: In [Noetic](https://github.com/ros/rosdistro/blob/72eeaec8e84f147feed051717ed324edc006a60c/noetic/distribution.yaml#L389), this key comes from a different repository (https://github.com/autoware-ai/messages.git).  What's the relationship between these repositories?

**Review by clalancette — CHANGES_REQUESTED** [review]

File: humble/distribution.yaml (hunk 1 above)
Comment (youtalk) [reply humble/distribution.yaml:658]: It was old version of the `autoware_msgs`. The ROS 1 Autoware.AI is no longer supported. That's why I removed it.
https://github.com/autowarefoundation/autoware?tab=readme-ov-file#using-autowareai

File: noetic/distribution.yaml [hunk 2]
```diff
@@ -383,31 +383,6 @@ repositories:
...
-      - vector_map_msgs
-      tags:
-        release: release/noetic/{package}/{version}
-      url: https://github.com/autoware-ai/messages-release.git
-      version: 1.14.0-1
-    source:
-      type: git
-      url: https://github.com/autoware-ai/messages.git
-      version: master
-    status: maintained
```
Comment (cottsay) [inline noetic/distribution.yaml:410]: It looks like there are downstream dependencies of at least one of these packages already in the rosdistro index.

Might I suggest, as an alternative to pulling the packages out of the index, you flip the `status` field to `end-of-life` and leave the packages alone?

Comment (youtalk) [reply noetic/distribution.yaml:410]: Thank you for your suggestion. I've flipped it instead. https://github.com/ros/rosdistro/pull/41122/commits/f293a01c54fb579c4244854e7ee528bab2034295

**Comment by youtalk** [comment]
@nuclearsandwich Please review it again. I hope it's mergable.

**Comment by youtalk** [comment]
@clalancette Could you review again? I've changed the status to `end-of-life`. https://github.com/ros/rosdistro/pull/41122/commits/f293a01c54fb579c4244854e7ee528bab2034295

**Pull request closed without merging**